README.md

```diff
@@ -355,6 +355,16 @@ class Post
 end
 ```
 
+You can also explicitly declare which policy to use for a specific instance of a class.
+
+```ruby
+class Post
+  def policy_class
```

Comment:
I'm fairly sure the proper way to do this is to define `policy_class` as a class method like so:

```ruby
class Post
  def self.policy_class
    ...
```

The reason is you can pass the class to the pundit helper methods - `authorize(Post)` for example.